ipie/walkers/uhf_walkers.py

```diff
@@ -158,6 +158,42 @@ def reortho_batched(self):
 
         return self.detR
 
+    def reortho_fp(self):
```

Comment:
Again, I think this should live somewhere else.

Reply:
done